### ISSUE TITLE
INT Sales Manager > Lead  Cannot Open the Page

### ISSUE THREAD

**Issue by SherryXLiu** [Fujitsu-America/Fujitsu-Repository#39]
**Describe the bug**
INT Sales Manager > Lead  Cannot Open the Page

**Instance Name**
Instance:
dpqs2i
**Responsibility Name**
Responsibility:
Sales Manager
**Navigation**
Navigation:

**To Reproduce**
![image](https://user-images.githubusercontent.com/85246030/121422142-ea3cb580-c93c-11eb-89f9-8512ef89580b.png)

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by SherryXLiu** [comment]
All these functions get Error exception Customer and Contact
![image](https://user-images.githubusercontent.com/85246030/121422961-c29a1d00-c93d-11eb-9a38-54113fd7bd00.png)

**Comment by vinaytalasila** [comment]
Sherry,

We will look into this and keep you posted.

**@suhas-deshpande** : Could you please check what technical / functional setups are missing related to this issue in INT 12.2.10 vs STG 12.2.10 and apply them in INT 12.2.10. If there are any configurations that need to be done by DBA team then please add Amit and Somu to this ticket.

Thanks
Vinay Talasila

**Comment by suhas-deshpande** [comment]
Hi Sherry - Kindly test now. You should be able to open the page now. You resource id was not mapped to your user in DPQS2I which has now been fixed.

Thanks,
Suhas

**Comment by SherryXLiu** [comment]
@suhas-deshpande : Yes. It works now. 
Could you provide the fix screenshot? I would understand why it is occurred and if other users could face same problem 
Thanks

**Comment by suhas-deshpande** [comment]
@SherryXLiu - While debugging various issues In INT, I had attached my user with your resource from below navigation to be able to access the forms and responsibilities -

CRM resource Manager - Maintain Resources - Resources.

Due to this you got the error of invalid resource id. I have now attached your user correctly and going forward I will not need to change it as I already have mine in place.

Thanks,
Suhas
